internal/runbits/requirements/requirements.go

```diff
@@ -117,15 +117,15 @@ type Requirement struct {
 	BitWidth      int // Only needed for platform requirements
 	Namespace     *model.Namespace
 	NamespaceType *model.NamespaceType
-	Operation     bpModel.Operation
+	Operation     types.Operation
```

Comment:
Honestly I'm not a fan of either. I try to avoid underscores and camelcase in package names as it's not idiomatic, but that then creates these scenarios where we have to alias package names at import times. Maybe this is another area where we give idiomatic go the finger..

I'll file a followup ticket for us to consider. I won't touch this code as is because it's used in a lot of places and it feels like a nit. Refactoring tools don't let you refactor the alias a package gets imported as, so it'd be a bit of a pain to change just to then change it again based on what we might decide in the ticket I'm filing.

Reply:
DX-2790